### PR TITLE
Implement ads displaying

### PR DESCRIPTION
**Pull request by demoronator**

### CODE REVIEW

File: src/app/ads/display/display.component.html [hunk 1]
```diff
@@ -1 +1,162 @@
...
+<html lang="en">
+  <head>
+    <!-- TODO: include head -->
+  </head>
+  <header>
+    <!-- TODO: include navbar -->
+    <link rel="stylesheet" href="/stylesheets/style.css" type="css" />
+  </header>
+
+  <body>
```
Comment (EdmundLT) [inline src/app/ads/display/display.component.html]: everything above the body is not required, change body tag to div tag

File: src/app/services/ads.service.ts [hunk 2]
```diff
@@ -2,18 +2,31 @@ import { HttpClient } from '@angular/common/http';
...
+  constructor(private http: HttpClient) { }
   //Get ALL ads from database
   getAds(): Observable<any> {
     return this.http.get(`${environment.apiUrl}/${this.baseUrl}/all`);
   }
   getAd(adsId: string): Observable<any> {
-    return this.http.get(`${this.baseUrl}/${adsId}`);
+    return this.http.get(`${environment.apiUrl}/${this.baseUrl}/${adsId}`);
+  }
+  disableAd(adsId: string): Observable<any> {
```
Comment (EdmundLT) [inline src/app/services/ads.service.ts:20]: should be post method

File: src/app/services/ads.service.ts [hunk 3]
```diff
@@ -2,18 +2,31 @@ import { HttpClient } from '@angular/common/http';
...
+    // TODO: Authentication
+    return this.http.get(`${environment.apiUrl}/${this.baseUrl}/disable/${adsId}`);
+  }
+  activateAd(adsId: string): Observable<any> {
+    // TODO: Authentication
+    return this.http.get(`${environment.apiUrl}/${this.baseUrl}/activate/${adsId}`);
```
Comment (EdmundLT) [inline src/app/services/ads.service.ts]: should be post method

**Review by EdmundLT — COMMENTED** [review]
Hi Wonyoung, please check the "Conversation", i made some comments

thanks a lot

**Comment by demoronator** [comment]
Fixed the above comments.
For the API methods, we should change the backend too, because they are GET.
Thank you!

**Review by EdmundLT — APPROVED** [review]
LGTM

**Pull request merged**